src/components/POIForm.component.js

```diff
@@ -59,6 +50,7 @@ class POIForm extends Component {
     }
     const poi = {
       title: name,
+      map_by_year: selectedMap.year,
```

Comment:
We should use standard camelCase here, and use an adapter in the api wrapper to convert it to the api naming convention